main.js

```diff
@@ -39,6 +39,10 @@ const LANGUAGES = {
   'scala': {
     file: 'seeds/scala_seed.scala',
     class: 'scala'
+  },
+  'csharp': {
```

Comment:
Alphabetical order here as well - for developers to find what they're looking for and to prevent merge conflicts.

Reply:
Fixed!